src/components/CreateLinkIcon.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { css } from 'glamor';
+import { CreateIcon } from './SvgIcons';
+
+const CreateLinkIcon = props => (
```

Comment:
`props` aren't currently being used.

That being said, I think this component would probably be slightly nicer if it was just `CreateLink`. Then it could accept a single prop, `to` 😄

Reply:
I agree!